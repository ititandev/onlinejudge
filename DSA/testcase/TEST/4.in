10
-26 -49.98 -47.73 4.61 -72.69 72.44 95 -25.17 -76.64 23.75
