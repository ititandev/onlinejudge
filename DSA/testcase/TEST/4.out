  -76.64  -72.69  -49.98  -47.73     -26
  -25.17    4.61   23.75   72.44      95
